networking/networking.go

```diff
@@ -123,6 +136,53 @@ func Setup(podRoot string, podID types.UUID, fps []ForwardedPort, netList common
 	return &n, nil
 }
 
+// enableDefaultLocalnetRouting enables the route_localnet attribute on the supposedly default network interface.
+// This allows the setup of a loopback NAT to access port forwardings to Pods on the host with the localhost address.
```

Comment:
These lines are kinda long.

Also, the second sentence is a bit confusing. Maybe:

> This allows setting up a loopback NAT so the host can access pods' forwarded ports on the localhost address.